tests/database_metrics/test_parsers.py

```diff
@@ -0,0 +1,18 @@
+"""Tests for parsers.py"""
```

Comment:
note that this repo uses python's built-in unittest (through a wrapper called py3tester), which is not the testing framework used by the indicators repo. so i don't think this is going to pass, which will cause auto-deploy to fail.

Reply:
some recent unit tests as a sample here: https://github.com/cmu-delphi/operations/blob/main/tests/screenshots/test_utils.py

docs on running unit tests: https://github.com/cmu-delphi/operations/blob/main/tests/screenshots/test_utils.py

tldr:

```
docker run --rm delphi_python \
  python3 -m undefx.py3tester.py3tester --color repos/delphi/operations/tests
```

Reply:
Thanks for calling this out, didn't realize that was the case. I'll follow the same pattern as the other tests and ping you once that's done.

Reply:
no problem. as far as i know, indicators is the only delphi python code thats _isn't_ tested with unittest.

(marginally related side note: indicators is also the only delphi python code that's indented with 4 vs 2 spaces hah.. but not a big deal.)

Reply:
Should be updated now

Reply:
nice! i haven't run the tests myself, but if they pass for you that's good enough for me.

driveby tip: in case you didn't know, you can add `--full` to the line that runs tests, and it'll show coverage info on the command line. helps me write tests for all cases.